src/utils.h

```diff
@@ -301,6 +301,8 @@ ts_get_relation_relid(char const *schema_name, char const *relation_name, bool r
 	}
 }
 
+struct Hypertable;
```

Comment:
Is this necessary?

Reply:
Does not seem necessary.

Seems to be a leftover from merging commits with both main separately and rebasing this pull request on main.